recipes/eigen/run_test.bat

```diff
@@ -0,0 +1,3 @@
+C:\MinGW\bin\g++ -I%LIBRARY_INC% -o test test.cc
```

Comment:
Could someone explain to me how to get this compiled? Either through `g++` or some standard Windows mechanism. As it's just a test, it shouldn't matter which one is picked as long as it does compile.